Tests/Marketplace/marketplace_services.py

```diff
@@ -2905,10 +2923,41 @@ def _changelog_entry_bc_versions(predecessor_version: Version, rn_version: Versi
         return {str(bc_ver): bc_version_to_text.get(str(bc_ver)) for bc_ver in breaking_changes_versions if
                 predecessor_version < bc_ver <= rn_version}
 
+    def get_version_to_pr_numbers(self, release_notes_dir) -> Dict[str, List[int]]:
+        """
+        Get Dict[Version,List[PullRequests]] for the given directory
+        Args:
+            release_notes_dir: the directory to find the pull request numbers for
+
+        Returns:
+            a dict of version -> List[Pull Request Numbers] for each file in the release_notes_dir
+        """
+
+        def file_path(file):
+            return f"Packs/{self.name}/{self.RELEASE_NOTES}/{file}"
+
+        if not os.path.exists(release_notes_dir):
+            return {}
+        return {underscore_file_name_to_dotted_version(file): get_pull_request_numbers_from_file(file_path(file))
+                for file in filter_dir_files_by_extension(release_notes_dir, '.md')}
+
 
 # HELPER FUNCTIONS
 
 
+def get_pull_request_numbers_from_file(file_path) -> List[int]:
+    """
+    Uses git and regex to find the pull request numbers for the given file
+    Args:
+        file_path: The file to find PRs for
+
+    Returns:
+        A list of relevant pull request numbers for the given file
+    """
+    log_info: str = git.Git(CONTENT_ROOT_PATH).log(file_path)
+    return [int(i) for i in re.findall(PULL_REQUEST_PATTERN, log_info)]
```

Comment:
```suggestion
    return [int(pr_number) for pr_number in re.findall(PULL_REQUEST_PATTERN, log_info)]
```
Also, why do we need to cast to int? Is there a reason why we need the pr ids as numbers and not strings?

Reply:
doesnt really need to be. Ill change it